reactioncommerce/reaction-next-starterkit
# Checkout complete page

Resolves #244 
Impact: **minor**
Type: **feature**

## Issue

Add a checkout complete page

## Solution

Added a checkout complete page using many existing components from cart and checkout.

## Breaking changes

none

## Testing

1. In Reaction, sign in, add items to cart and checkout
1. Run through the checkout process and create an order
1. After the order is created, open Robo3T and look for the newly created order
1. Copy the order id and base64 encode it as `reaction/order:<order-id>`
1. In the starter kit, ensure you're signed in as the same user as reaction with the `meteor-login-token`
1. Then navigate to `http://localhost:4000/checkout/order/<order-id>`
1. See the checkout completed page with all relevant data
